proto/controlclient.proto

```diff
@@ -60,8 +61,30 @@ message _ListCachesRequest {
   string next_token = 1;
 }
 
+message _CacheLimits {
+  // The amount of transactions per second that can be exercised
+  uint32 max_traffic_rate = 1; 
+  // The amount of traffic per second that can be exercised in KiB
+  uint32 max_throughput_kbps = 2;
+  // The maximum size of a single item in KiB
+  uint32 max_item_size_kb = 3;
+  // The maximum TTL allowed for a single item, in seconds
+  uint64 max_ttl_seconds = 4;
+}
+
+message _TopicLimits {
+  // The amount of messages that can be published per second
+  uint32 max_publish_rate = 1;
```

Comment:
do you know if this is _per_ topic, or across all topics?

Reply:
This is _per cache_, so across all topics